src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/MessageProcessor.java

```diff
@@ -31,61 +24,27 @@
 @Slf4j
 public class MessageProcessor {
 
-    @Value("${case-feed-future-date-offset}")
-    private int caseFeedFutureDateOffset;
-
-    @SuppressWarnings("UnstableApiUsage") // Not part of the final product
+    @SuppressWarnings("UnstableApiUsage")
     private final EventBus eventBus;
 
-    private final GatewayMessageParser parser;
-
     private final TelemetryService telemetryService;
 
     private final CourtCaseService courtCaseService;
 
     @Autowired
-    public MessageProcessor(GatewayMessageParser gatewayMessageParser,
-                            EventBus eventBus,
+    public MessageProcessor(EventBus eventBus,
                             TelemetryService telemetryService,
                             CourtCaseService courtCaseService) {
         super();
-        this.parser = gatewayMessageParser;
         this.eventBus = eventBus;
         this.telemetryService = telemetryService;
         this.courtCaseService = courtCaseService;
     }
 
-    public void process(String message) {
-        parse(message).ifPresent(this::process);
-    }
-
-    private Optional<MessageType> parse(String message) {
```

Comment:
Parsing moved to message receiver because there are now 2 receivers - one for each type of message. MessageProcessor now just processes an ExternalDocumentRequest